doc/rfc/0000-rfc-repository.md

```diff
@@ -0,0 +1,168 @@
+# RFC 0000 - RFC repository
+
+- Date proposed: 2023-02-28
+- Date approved: under review
+
+## Summary
+
+This RFC proposes storing OpenTitan RFCs as files in a Git repo, and using
+an open source development workflow for the RFC process. RFCs would be
+stored in the RFC repo as text files. RFCs would be proposed, reviewed, and
+accepted/rejected using GitHub pull requests (PRs). The goal is to ensure
+project decisions are better documented, and to make the RFC process more
+visible and transparent. The workflow proposed here will already be familiar to
+open source contributors and should add little overhead.
+
+## Motivation
+
+RFCs in OpenTitan are used for decisions that need consensus from the technical
+committee. The RFC workflow before this proposal is roughly:
+
+1. Write up your RFC somewhere.
+2. Create an issue labelled `RFC:Proposal` linking to your RFC in the main repo.
+3. Receive and address comments from others.
+4. Apply the `RFC:TC review` label when the RFC is ready to be voted on.
+5. The technical committee will vote on pending RFCs at their next meeting.
+5. Approved RFCs have their label changed to `RFC:Approved`.
+6. Rejected RFCs have their label removed.
+
+Typically, authors write their RFCs in one of two places:
+
+* Google Docs.
+* The GitHub issue's body as Markdown.
+
+The entry-point for finding RFCs stored like this is through the `RFC:Approved`
+label in GitHub issues. The OpenTitan repository gets many issues and has many
+labels to sort them. Discovering RFCs requires regularly checking the `RFC`
+labels or keeping up to date with OpenTitan's fast moving issues.
+
+Allowing authors to choose where they store their RFC content makes it
+harder to reference them as documentation. It also means we can't manage them
+consistently: we have no public-by-default rule for Google Docs, leading to
+many RFCs restricted only to select people. Comments are left in both GitHub
+iAssue comments and Google Docs reviews.
+
+In summary, the motivation of this RFC is to have:
+
+1. a single store of RFC documents,
+2. where RFCs can easily be discovered,
+3. where RFC content can be enforced to be public,
+4. where RFC comments are given and addressed in a consistent workflow.
```

Comment:
Suggest to slightly reformat to improve legibility
```suggestion
In summary, the motivation of this RFC is to have a single store of RFC documents, where

* RFCs can easily be discovered,
* RFC content can be enforced to be public,
* RFC comments are given and addressed in a consistent workflow.
```

Reply:
Thanks, this is much clearer. Updated.